.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: android
+android:
+  components:
+    - tools
+    - platform-tools
+
+    # The SDK version used to compile this project
+    - android-28
+
+jdk:
+  - openjdk8
+before_install:
+  - yes | sdkmanager "platforms;android-28"
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
+    - $HOME/.android/build-cache
+script:
+  - "./gradlew clean build test"
```

Comment:
We should run the espresso tests as well.

Reply:
There are no espresso tests for now, so the job will fail due to no tests would be available

Reply:
There are espresso tests. You can create a follow up ticket if you'd prefer that route.